packages/libs/src/sdk/api/users/types.ts

```diff
@@ -77,3 +77,13 @@ export const SendTipSchema = z
   .strict()
 
 export type SendTipRequest = z.input<typeof SendTipSchema>
+
+export const SubmitReactionRequestSchema = z.object({
+  userId: HashId,
+  metadata: z.object({
+    reactedTo: z.string().nonempty(),
+    reactionValue: z.number().int().min(1).max(4)
```

Comment:
nit: the caller shouldn't have to know about the enum values - let's map them for the end caller similar to how we do in the client:

https://github.com/AudiusProject/audius-protocol/blob/5e026a1c05c1a69a620de0aaa170814565b5f70a/packages/common/src/store/ui/reactions/types.ts#L4-L11

https://github.com/AudiusProject/audius-protocol/blob/5e026a1c05c1a69a620de0aaa170814565b5f70a/packages/common/src/store/ui/reactions/utils.ts#L3-L8

https://github.com/AudiusProject/audius-protocol/blob/3c898a8a97b0389ec489d906a42693532e1787c5/packages/web/src/common/store/ui/reactions/sagas.ts#L87

Reply:
cool i'll add this! thanks